### PR TITLE
Urwa/landing page

### PR DESCRIPTION
**Pull request by UrwaIhtesham**
**Frontend**
- Changed docker file of frontend
- Added media queries to make landing page responsive
- Changed case sensitivity in home.js

### CODE REVIEW

**Review by muhammadqasim21 — APPROVED** [review]

**Comment by SabaSaleem5317** [comment]
yayyy

**Review by SabaSaleem5317 — APPROVED** [review]

**Review by anssidhu — APPROVED** [review]
Looks good

File: frontend/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Stage 1: Build the React application
-FROM node:14 AS build
+FROM node:16 AS build
```
Comment (azubair-experlabs) [inline frontend/Dockerfile:2]: Is the docker working now on the frontend? And why not the latest node like 20, 21? Was there any dependency issues?

File: frontend/public/images/pets.jpg
Comment (azubair-experlabs) [inline frontend/public/images/pets.jpg:1]: One quick tip to make your apps faster is to compress your images before adding in code for example keep a target to keep every image asset less than 1 MB or 500 KB. there are multiple online tools that do lossless compression( compress without losing the quality) like https://compressor.io/ or https://compresspng.com/

File: frontend/src/App.js [hunk 2]
```diff
@@ -28,6 +29,7 @@ function App() {
         
     </div>*/
     <Home/>
+    //<LandingPage/>
```
Comment (azubair-experlabs) [inline frontend/src/App.js:32]: Are these comments intentional?

**Review by azubair-experlabs — APPROVED** [review]

File: frontend/Dockerfile (hunk 1 above)
Comment (UrwaIhtesham) [reply frontend/Dockerfile:2]: yes it is working now. 
i will update the node version in the next pull request

File: frontend/public/images/pets.jpg
Comment (UrwaIhtesham) [reply frontend/public/images/pets.jpg:1]: ok thank you. i will look into it.

File: frontend/src/App.js (hunk 2 above)
Comment (UrwaIhtesham) [reply frontend/src/App.js:32]: yes i did it intentionally. will remove them after testing

**Pull request merged**